Source/Core/Core/PowerPC/JitArm64/JitArm64_BackPatch.cpp

```diff
@@ -127,10 +127,25 @@ bool JitArm64::DisasmLoadStore(const u8* ptr, u32* flags, ARM64Reg* reg)
 	return false;
 }
 
-u32 JitArm64::EmitBackpatchRoutine(ARM64XEmitter* emit, u32 flags, bool fastmem, bool do_padding, ARM64Reg RS, ARM64Reg addr)
+u32 JitArm64::EmitBackpatchRoutine(u32 flags, bool fastmem, bool do_farcode,
+	ARM64Reg RS, ARM64Reg addr,
+	BitSet32 gprs_to_push, BitSet32 fprs_to_push)
 {
+	bool in_far_code = false;
 	u32 trouble_offset = 0;
-	const u8* code_base = emit->GetCodePtr();
+	const u8* trouble_location = nullptr;
+	const u8* code_base = GetCodePtr();
+
+	SlowmemHandler handler;
```

Comment:
Why is this up here? Unless I missed something in this Diff, it isn't used until line 239 (and only within that ```if``` block).